### PR TITLE
Xref Test

### PR DESCRIPTION
**Pull request by elbrujohalcon**

### CODE REVIEW

File: src/org.erl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        ]).
+
+-spec start() -> {ok, _} | {error, term()}.
+start() -> application:ensure_all_started(org).
+
+-spec stop() -> ok | {error, term()}.
+stop() -> application:stop(org).
+
+-spec start(atom(), any()) -> {ok, pid()} | {error, term()}.
+start(normal, _Args) ->
```
Comment (gadgetci) [inline src/org.erl]: According to **Xref**:
> `org:start/2` calls undefined function `lager_config:get/2`

File: src/org.erl [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  lager:alert("Application ~p starting", [?MODULE]),
+  {ok, self()}.
+
+-spec stop(atom()) -> ok.
+stop(_State) ->
```
Comment (gadgetci) [inline src/org.erl]: According to **Xref**:
> `org:stop/1` calls undefined function `lager:md/0`

Comment (gadgetci) [inline src/org.erl]: According to **Xref**:
> `org:stop/1` calls undefined function `lager_config:get/2`

File: src/org.erl (hunk 1 above)
Comment (gadgetci) [inline src/org.erl]: According to **Xref**:
> `org:start/2` calls undefined function `lager:md/0`

Comment (gadgetci) [inline src/org.erl]: According to **Xref**:
> `org:start/2` calls undefined function `lager:do_log/9`

File: src/org.erl (hunk 2 above)
Comment (gadgetci) [inline src/org.erl]: According to **Xref**:
> `org:stop/1` calls undefined function `lager:do_log/9`

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+dep_katana = git git://github.com/inaka/erlang-katana 0.2.4
+
+DIALYZER_DIRS := ebin/
+DIALYZER_OPTS += --verbose --statistics -Werror_handling \
+                 -Wrace_conditions
+
+ERLC_OPTS ?= +debug_info +warn_export_vars +warn_shadow_vars \
+	+warn_obsolete_guard # +bin_opt_info +warn_export_all +warn_missing_spec
+ERLC_OPTS += +'{parse_transform, lager_transform}' +debug_info
+
```
Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding commands for target `shell'

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+include erlang.mk
```
Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding commands for target `shell'

**Comment by gadgetci** [comment]
According to **Compiler**:
> warning: ignoring old commands for target `shell'

File: Makefile (hunk 4 above)
Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding commands for target `shell'

**Comment by gadgetci** [comment]
According to **Compiler**:
> warning: ignoring old commands for target `shell'

File: Makefile (hunk 4 above)
Comment (gadgetci) [inline Makefile]: According to **Dialyzer**:
> warning: overriding commands for target `shell'

**Comment by gadgetci** [comment]
**dialyzer** failed with exit status: 2

**Comment by gadgetci** [comment]
According to **Dialyzer**:
> warning: ignoring old commands for target `shell'

File: Makefile (hunk 4 above)
Comment (gadgetci) [inline Makefile]: According to **Dialyzer**:
> warning: overriding commands for target `shell'

**Comment by gadgetci** [comment]
According to **Dialyzer**:
> warning: ignoring old commands for target `shell'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

File: Makefile (hunk 4 above)
Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

Comment (gadgetci) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> warning: ignoring old recipe for target 'shell'

File: Makefile (hunk 4 above)
Comment (gadgetci-staging) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

Comment (gadgetci-staging) [inline Makefile]: According to **Dialyzer**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

File: Makefile (hunk 4 above)
Comment (gadgetci-staging) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

File: Makefile (hunk 4 above)
Comment (gadgetci-staging) [inline Makefile]: According to **Dialyzer**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001496-0431100-0544272/.org.plt' failed

**Comment by gadgetci-staging** [comment]
**dialyzer** failed with exit status: 2

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001496-0669679-0096819/.org.plt' failed

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

File: Makefile (hunk 4 above)
Comment (gadgetci-staging) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

Comment (gadgetci-staging) [inline Makefile]: According to **Dialyzer**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
**dialyzer** failed with exit status: 2

File: Makefile (hunk 4 above)
Comment (gadgetci-staging) [inline Makefile]: According to **Dialyzer**:
> warning: overriding recipe for target 'shell'

Comment (gadgetci-staging) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
According to **Elvis**:
> Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by gadgetci-staging** [comment]
**dialyzer** failed with exit status: 2

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> warning: ignoring old recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001497-0374434-0903756/.org.plt' failed

**Comment by gadgetci-staging** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

File: Makefile (hunk 4 above)
Comment (gadgetci-staging) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

Comment (gadgetci-staging) [inline Makefile]: According to **Compiler**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Compiler**:
> warning: ignoring old recipe for target 'shell'

File: Makefile (hunk 4 above)
Comment (gadgetci-staging) [inline Makefile]: According to **Dialyzer**:
> warning: overriding recipe for target 'shell'

Comment (gadgetci-staging) [inline Makefile]: According to **Dialyzer**:
> warning: overriding recipe for target 'shell'

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001497-0375080-0523307/.org.plt' failed

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001497-0376486-0883145/.org.plt' failed

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001497-0379430-0908061/.org.plt' failed

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001497-0403264-0026004/.org.plt' failed

**Comment by gadgetci-staging** [comment]
According to **Dialyzer**:
> recipe for target '/tmp/gadget/gadget-test/org-repo/0001497-0413223-0365784/.org.plt' failed

**Review by elvisci — DISMISSED** [review]
Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — DISMISSED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Review by elvisci — CHANGES_REQUESTED** [review]
- Dependency 'lager' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'eper' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'
- Dependency 'katana' is not using appropriate protocol, please change this to something like '(https://.*|[0-9]+([.][0-9]+)*)'

**Comment by elbrujohalcon** [comment]
This PR is useless now 😢

**Pull request closed without merging**